### DISCUSSION TITLE
Add a note about goals on a brand new dashboard

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1833] [Anything Else]
"I had no idea this feature even existed. would be nice to make this a bit more discoverable! Perhaps you could show a discrete box with "Define some goals" in that spot instead of hiding it entirely when there are no goals?"

(feedback via [twitter](https://twitter.com/fwouts/status/1514226221219217413))

**Comment by ukutaht** [comment]
I think that would be a great addition. The notice should probably disappear after some amount of 'onboarding time', though. For those who don't use goals at all, could be seen as a distraction or noise.

**Reply by metmarkosaric** [reply]
yeah for sure. something like that email report banner that we have on the top would work fine. show it at the start, allow you to close it or make it never appear when there's one goal in the dashboard
